lang/interpret_test.go

```diff
@@ -1130,6 +1138,7 @@ func TestAstFunc2(t *testing.T) {
 			errs: errs{
 				fail1: fail1,
 				fail9: fail9, // TODO: rename
+				fail8: fail8, // TODO: rename
```

Comment:
`failInterpolate`?

Reply:
I'll send a cleanup patch. This would have been smarter :P